packages/grid/_modules_/grid/components/RowElements.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { CellIndexCoordinates, Columns, GridOptions, RenderContextProps, RowModel } from '../models';
+import { LeftEmptyCell, RightEmptyCell } from './Cell';
+import { Row } from './Row';
+import { RowCells } from './RowCells';
+
+export const RowElements: React.FC<{
+  row: RowModel;
+  domIndex: number;
+  renderContext: RenderContextProps;
+  options: GridOptions;
+  columns: Columns;
+  cellFocus: CellIndexCoordinates | null;
+  hasScroll: { y: boolean; x: boolean };
+  selected: boolean;
+  rowHeight: number
+}> = React.memo(({
+  row,
+  domIndex,
+  renderContext,
+  options,
+  columns,
+  cellFocus,
+  hasScroll,
+  selected,
+  rowHeight,
+}) => {
+  return (
+    <Row
+      className={
+        (renderContext.firstRowIdx + domIndex) % 2 === 0 ? 'Mui-even' : 'Mui-odd'
+      }
+      id={row.id}
+      selected={selected}
```

Comment:
Maybe we can remove React.memo() from RowCells by moving selected state lookup to Row element itself, but I'm not familiar with row selection in this library.